_data/accessibility-tests/text-input.yml

```
@@ -0,0 +1,72 @@
+title: Text input
+component_status: pass
+test_items:
+# General tests
+  - summary: Instructions that use visual cues also use non-visual information.
+    summary_additional: |
+      Text input instructions are announced through an ARIA label and can be understood 
+      by users who can’t see shapes or locations.
```

The additional summary reads to me like a screen reader test since it is referencing the aria-label. Just flagging this to confirm that the additional instructions only appear in the aria-label, not in the visual text. If so, should this be a screen reader test instead of general?

Alternatively, I wonder if 1.3.3 is speaking to all written instruction. If so, maybe something like this  suggestion would work: 
```suggestion
  - summary: Instructions that use visual cues also provide non-visual instruction.
    summary_additional: |
       If you use visual cues like shapes or locations to describe controls (e.g., "input below"), also provide extra instructions. This way, users who can't see shapes or locations can still understand how to use the controls.
```
Apologies for missing the [original conversation](https://github.com/uswds/uswds-site/pull/2697#discussion_r1619373597)!